src/ML-SNAP/compute_sna_grid.h

```diff
@@ -0,0 +1,56 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   https://www.lammps.org/ Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifdef COMPUTE_CLASS
+// clang-format off
+ComputeStyle(sna/grid,ComputeSNAGrid);
+// clang-format on
+#else
+
+#ifndef LMP_COMPUTE_SNA_GRID_H
+#define LMP_COMPUTE_SNA_GRID_H
+
+#include "compute_grid.h"
+
+namespace LAMMPS_NS {
+
+class ComputeSNAGrid : public ComputeGrid {
+ public:
+  ComputeSNAGrid(class LAMMPS *, int, char **);
+  ~ComputeSNAGrid() override;
+  void init() override;
+  void init_list(int, class NeighList *) override;
```

Comment:
There doesn't seem to be a neighbor list request coming from this compute, so the `init_list()` member function and the list pointer would be obsolete (and the neigh_list.h header not required).

Reply:
Done